blazor/datagrid/filtering.md

```diff
@@ -120,4 +120,43 @@ GreaterThanOrEqual |Checks whether a value is greater than or equal to the speci
 LessThan |Checks whether the value is less than the specified value. |Number &#124; Date
 LessThanOrEqual |Checks whether the value is less than or equal to the specified value. |Number &#124; Date
 
-> By default, the **Operator** value is **Equal** in [Columns](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridFilterSettings.html#Syncfusion_Blazor_Grids_GridFilterSettings_Columns) property of **GridFilterSettings** component.
\ No newline at end of file
+> By default, the **Operator** value is **Equal** in [Columns](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridFilterSettings.html#Syncfusion_Blazor_Grids_GridFilterSettings_Columns) property of **GridFilterSettings** component.
+
+## Get filtered records when using remote data source
+
+Using the [GetFilteredRecordsAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_GetFilteredRecordsAsync) method, you can retrieve the filtered records details from the Grid component. The `GetFilteredRecordsAsync` method returns the filtered records in the form of objects when using a remote data source. So, you need to deserialize the object to retrieve the filtered records.
```

Comment:
Use the [GetFilteredRecordsAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_GetFilteredRecordsAsync) method to retrieve the details of the filtered records from the Grid component. The 'GetFilteredRecordsAsync' method returns the filtered records in the form of objects when using a remote data source. So, you need to deserialize the object to retrieve the filtered records.